templates/player.mustache

```diff
@@ -30,5 +30,5 @@
     <iframe data-frameSrc="{{src}}" width="95%" height="455px" class="ocplayer" allowfullscreen="true"></iframe>
 {{/loggedin}}
 {{! Print the link to the video. }}
-<a style="display:block;" target="_blank" href="{{link}}">Zum Video</a>
+<a style="display:block;" target="_blank" href="{{link}}">{{#str}} watch_the_video, filter_opencast {{/str}}</a>
```

Comment:
Shouldn't that be to_the_video? (Sorry for the late review :see_no_evil: )

Reply:
Of course. Thanks for looking into this.

Reply:
I fixed it.